### PR TITLE
Bug 1793648, added disk recommendations for etcd

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1793648

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@rjhowe @mpbarrett @dmoessne @bniver PTAL

**Comment by rjhowe** [comment]
LGTM

**Comment by ahardin-rh** [comment]
We received approval from Mike Hackett in an email as well.

@geliu2016 Can you please provide a QE review? Thanks!

**Comment by bobfuru** [comment]
LGTM

**Comment by ahardin-rh** [comment]
@geliu2016 Can you please provide QE review?

**Comment by ahardin-rh** [comment]
@geliu2016 Can you please provide QE review?

**Comment by ahardin-rh** [comment]
Received QE approval in the BZ.

**Comment by ahardin-rh** [comment]
@dmoessne Is this change also relevant for 4.x?

File: install/prerequisites.adoc [hunk 1]
```diff
@@ -149,6 +149,10 @@ The size of storage depends on workload, the number of containers, the
 size of the running containers, and the containers' storage requirements.
 You must also configure storage to run containerized etcd.
 
+It is highly recommended that you use etcd with storage that handles small
+read/write operations quickly, such as SSD. Ceph, NFS, and spinning disks are
```
Comment (hexfusion) [inline install/prerequisites.adoc:153]: nit: `handles serial writes (fsync) quickly`. Most issues with disk are related to the speed at which fsync can persist state to disk and return.

**Pull request merged**